src/MLJNearestNeighborsInterface.jl

```diff
@@ -1,36 +1,129 @@
 module MLJNearestNeighborsInterface
 
+# ===================================================================
+# IMPORTS
 import MLJModelInterface
-import MLJModelInterface: MODEL_TRAITS
+import MLJModelInterface: @mlj_model, metadata_model, metadata_pkg,
+    Table, Continuous, Count, Finite, OrderedFactor, Multiclass
+import NearestNeighbors
+import StatsBase
+import Tables
 
-using ScientificTypes
-using MLJBase
-import MLJBase: @load
-import MLJBase: Table, Continuous, Count, Finite, OrderedFactor, Multiclass
+using Distances
+using FillArrays
+using LinearAlgebra
+using Statistics
 
-using Requires, Pkg, Pkg.TOML, OrderedCollections, Parameters
-using Tables, CategoricalArrays, StatsBase, Statistics
-import Distributions
+# ===================================================================
+## EXPORTS
+export list_kernels, ColumnTable, DictTable
 
+# Export KNN models
+export KNNClassifier, KNNRegressor, KnnKernel, MultitargetKNNRegressor
 
```

Comment:
Shouldn't `KnnKernel` be `KNNKernel`?

Reply:
`MultitargetKNNClassifier` appears to be missing in `export`. Which is strange as it does appear in my namespace.

Reply:
Yeah apparently models created with `@mlj_model` are automatically exported https://github.com/alan-turing-institute/MLJModelInterface.jl/blob/dev/src/model_def.jl#L186